install.sh

```diff
@@ -994,11 +994,24 @@ case "${SHELL}" in
     ;;
 esac
 
-cat <<EOS
+# show different instructions on adding Homebrew to path based on if the user has already done so
+if grep -qs "eval \"\$(${HOMEBREW_PREFIX}/bin/brew shellenv)\"" "${shell_profile}" && ! type brew &>/dev/null
```

Comment:
```suggestion
if grep -qs "eval \"\$(${HOMEBREW_PREFIX}/bin/brew shellenv)\"" "${shell_profile}" && type brew &>/dev/null
```

As this says `brew` was found.

Reply:
Removing the negation would make the condition evaluate as false since `brew` would not be found. I will change the prompt wording.

Reply:
I do not think this block is necessary because `brew` will already be in PATH. I will rewrite this logic